app/resources/1.1/material_artwork/sparql/premis_ie.sparql

```diff
@@ -30,6 +32,33 @@ CONSTRUCT {
     ?rep a premisowl:Representation;
         relSubType:rep ?ie .
 
+    ?evt a premisowl:Event, ?ev_type_iri ;
+        prov:startedAtTime ?ev_dateTime ;
+        prov:endedAtTime ?ev_dateTime ;
+        premisowl:note ?ev_detail ;
+        prov:used ?additionalProperty_iri ;
+        premisowl:outcome ?ev_outcome_iri ;
+        ?ev_agent_role ?ev_linking_agent .
+    
+    ?additionalProperty_iri a prov:Entity, ?additionalProperty_type_iri ;
+        rdf:value ?additionalProperty_value_simple ;
+        ?additionalProperty_prop_complex ?additionalProperty_value_complex .
+    
+    ?agent_organization a org:Organization ;
+        schema:name ?agent_organization_name ;
+        ?p_agent_organization ?o_agent_organization .
```

Comment:
Een `org:Organization` heeft een `schema:identifier` en `skos:prefLabel` nodig.

Zoiets?:
```
schema:identifier ?_agent_organization_id ;
skos:prefLabel ?agent_organization_label;
```

Skos nog toevoegen als prefix:
`PREFIX skos: <http://www.w3.org/2004/02/skos/core#>`

En nog:
`BIND(STRLANG(?agent_organization_name, "nl") AS ?agent_organization_label)`

Is die `schema:name` dan nog nodig?